guides/source/getting_started.md

```diff
@@ -1151,11 +1400,15 @@ end
 
 #### Extracting Partials
 
-We've already written a form for creating new products. Wouldn't it be nice if we could reuse that for edit and update? We can, using a feature called "partials" that allows you to reuse a view in multiple places.
+We've already written a form for creating new products. Wouldn't it be nice if
+we could reuse that for edit and update? We can, using a feature called
+"partials" that allows you to reuse a view in multiple places.
 
-We can move the form into a file called `app/views/products/_form.html.erb`. The filename starts with an underscore to denote this is a partial.
+We can move the form into a file called `app/views/products/_form.html.erb`. The
+filename starts with an underscore to denote this is a partial.
 
-We also want to replace any instance variables with a local variable, which we can define when we render the partial. We'll replace `@product` with `product`.
+We also want to replace any instance variables with a local variable, which we
+can define when we render the partial. We'll replace `@product` with `product`.
```

Comment:
Might be nice to give a brief explanation, for example:

We also want to replace any instance variables with a local variable so the partial can be used with different views and display different data that is passed to it.

This can be defined when we render the partial. We'll replace `@product` with `product`.

Reply:
Thanks @elhalvers. We discussed this a while back and didn't have a great explanation at the time. I can run with your suggestion. 👍